lib/routes/admin-api/metrics.test.js

```diff
@@ -165,7 +165,7 @@ test('should not delete unknown application', t => {
 
     return request
         .delete(`/api/admin/metrics/applications/${appName}`)
-        .expect(409);
+        .expect(404);
```

Comment:
:+1:

Reply:
After discussing some REST peculiarities, we've decided that this should be 200, since DELETE's should be idempotent; I'll fix this before merging